### PR TITLE
Hw25

### PR DESCRIPTION
**Pull request by andriylagun**

### CODE REVIEW

File: src/main/webapp/web.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        <servlet-name>index</servlet-name>
+        <servlet-class>com.internet.shop.controller.IndexController</servlet-class>
+    </servlet>
+
+    <servlet-mapping>
+        <servlet-name>index</servlet-name>
+        <url-pattern>/</url-pattern>
+    </servlet-mapping>
+
+</web-app>
```
Comment (tarkaiev) [inline src/main/webapp/web.xml]: no empty line at the end of file

Comment (andriylagun) [reply src/main/webapp/web.xml]: in intellij idea have empty line. Confused why here didn't

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 .idea/*
 *.iml
 target/*
+out/*
```
Comment (Sofasmile) [inline .gitignore]: add one empty line at the end of the file

File: pom.xml [hunk 3]
```diff
@@ -25,6 +25,13 @@
             <version>1.18.12</version>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```
Comment (Sofasmile) [inline pom.xml:28]: does project work without packagin war? and mvn plugin?

File: src/main/java/com/internet/shop/controller/IndexController.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package com.internet.shop.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet({"/","/home"})
```
Comment (Sofasmile) [inline src/main/java/com/internet/shop/controller/IndexController.java]: do we need to have two paths?

File: pom.xml (hunk 3 above)
Comment (andriylagun) [reply pom.xml:28]: yeah. My project is working without packagin war and mvn plugin. Here you can see:
зображення
зображення

File: src/main/java/com/internet/shop/controller/IndexController.java (hunk 4 above)
Comment (andriylagun) [reply src/main/java/com/internet/shop/controller/IndexController.java]: I think. this is test version. And i try take two paths to controller.  But will change that

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**